### ISSUE TITLE
share the weights for VOC?

### ISSUE THREAD

**Issue by studyharderer** [Stick-To/FCOS-tensorflow#4]
Dear author，thank you for sharing your reimplement of FCOS！
Can you share your trained weights on VOC ?

**Comment by studyharderer** [comment]
BTW, i train this code with VOC2007 train dataset (only 2502 pic) ，when it goes to epoch=140 ，mean loss = 2.8 ，and its performance on VOC2007 val dataset is not good ...   Is this normal？

**Comment by Stick-To** [comment]
Sorry,  I have't keeped my weight, you could test your weight on some photos
